### ISSUE TITLE
I believe the labled batched size for CIFAR-100 in your paper is wrong

### ISSUE THREAD

**Issue by psellcam** [ahmetius/LP-DeepSSL#2]
I tried the CIFAR-100 dataset using the parameters specified in the paper. LR =0.02 , B_L = 31 B=128  etc but the code performs very poorly in trainstage2. For CIFAR-100 should the parameter B_L be 31?

**Comment by ahmetius** [comment]
Hi,

Can you tell us what is the result you are getting in CIFAR-100?

**Comment by psellcam** [comment]
Just that the accuracy drops massively to around 30/40 percent and there is where it stays

I have much better results from reducing the initial LR from 0.2 to lower values and increasing the number of labeled samples per pseudo batch

**Comment by ahmetius** [comment]
What is the exact command that you are running? Have you waited until the training finished?

I just ran a new experiment with the command:

`python train_stage2.py --labeled-batch-size=31 --num-labeled=4000 --gpu-id=5 --label-split=10 --isMT=False --isL2=True --dataset=cifar100`

And the accuracy is 54.120, similar to what is reported in the paper. Note that the accuracy goes down initially in the second stage, but goes back up as the learning rate gets smaller. This can indeed be accelerated with a smaller initial learning rate.

**Comment by psellcam** [comment]
Thank you for the fast reponse. The initial decrease in accuracy had scared me
